PR-template.html

```diff
@@ -0,0 +1,6 @@
+- [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
+- [ ] Each commit in the pull request should have a meaningful subject line and body.
+- [ ] Format the pull request title like `[CAMEL-XXX] Fixes bug in camel-file component`, where you replace `CAMEL-XXX` with the appropriate JIRA issue.
+- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
+- [ ] Run `mvn clean install` in your module to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
+- [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).
```

Comment:
- the ICLA is not needed, usually it is something for contributors becoming committers or part of the PMCs.
- the point about JIRA issue is true only if the PR is large and introduce somenthing new, for little things is not needed.
- so sometimes the point 3 is not needed